### PR TITLE
Large file encryption

### PR DESCRIPTION
**Pull request by murali-shris**
**- What I did**
-added support for large file encryption in chunks
**- How I did it**
- added converters for AES encryption and decryption of file chunks
- added byte splitter to split files based on chunk size
- added new methods in encryption_service - encryptFileInChunks, decryptFileInChunks

**- How to verify it**
- run the unit test at_client/test/file_encryption_test.dart

### CODE REVIEW

File: at_client/lib/src/service/encryption_service.dart [hunk 1]
```diff
@@ -478,4 +481,50 @@ class EncryptionService {
...
     return encryptedValue;
   }
+
+  Future<File> encryptFileInChunks(
+      File inputFile, String fileEncryptionKey, int chunkSize) async {
+    var chunkedStream = ChunkedStreamReader(inputFile.openRead());
+    final length = inputFile.lengthSync();
+    var readBytes = 0;
+    final fileName = inputFile.uri.pathSegments.last;
+    final encryptedFile = File('${inputFile.parent.path}/encrypted_$fileName');
```
Comment (sarika01) [inline at_client/lib/src/service/encryption_service.dart]: '/' needs to be replaced with Platform.pathSeparator.

File: at_client/lib/src/service/encryption_service.dart [hunk 2]
```diff
@@ -478,4 +481,50 @@ class EncryptionService {
...
+  }
+
+  Future<File> decryptFileInChunks(
+      File encryptedFile, String fileDecryptionKey, int chunkSize) async {
+    var chunkedStream = ChunkedStreamReader(encryptedFile.openRead());
+    var startTime = DateTime.now();
+    final length = encryptedFile.lengthSync();
+    final fileName = encryptedFile.uri.pathSegments.last;
+    var readBytes = 0;
+    final decryptedFile =
```
Comment (sarika01) [inline at_client/lib/src/service/encryption_service.dart]: need to use Platform.pathSeparator

File: at_client/lib/src/service/encryption_service.dart (hunk 1 above)
Comment (murali-shris) [reply at_client/lib/src/service/encryption_service.dart]: done

File: at_client/lib/src/service/encryption_service.dart (hunk 2 above)
Comment (murali-shris) [reply at_client/lib/src/service/encryption_service.dart]: done

**Review by sarika01 — APPROVED** [review]

**Pull request merged**